docs/source/deployment/azure_container_instance.rst

```diff
@@ -421,10 +454,10 @@ Validate Azure container instance with sample data POST request
 
 .. code-block:: bash
 
-    > curl -X \
-      POST "http://bentomlirisclassifier777.eastus.azurecontainer.io:5000/predict" \
-      --header "Content-Type: application/json" \
-      -d '[[5.1, 3.5, 1.4, 0.2]]'
+    $ curl -X \
+        POST "http://bentomlirisclassifier777.eastus.azurecontainer.io:5000/predict" \
```

Comment:
could this line be 
```
POST $(az container show --resource-group iris-classifier --name bentomlirisclassifier --query ipAddress.fqdn)
```
?

no need to change now.. maybe for another time when you get a chance to verify if it works

Reply:
yeah, maybe next time when we have a chance verify it